bundle/manifests/ptp-operator.clusterserviceversion.yaml

```diff
@@ -370,20 +370,22 @@ spec:
                 - name: OPERATOR_NAME
                   value: ptp-operator
                 - name: RELEASE_VERSION
-                  value: v4.13.0
+                  value: v4.14.0
                 - name: LINUXPTP_DAEMON_IMAGE
-                  value: quay.io/openshift/origin-ptp:4.13
+                  value: quay.io/openshift/origin-ptp:4.14
                 - name: KUBE_RBAC_PROXY_IMAGE
-                  value: quay.io/openshift/origin-kube-rbac-proxy:4.13
+                  value: quay.io/openshift/origin-kube-rbac-proxy:4.14
                 - name: SIDECAR_EVENT_IMAGE
-                  value: quay.io/openshift/origin-cloud-event-proxy:4.13
+                  value: quay.io/openshift/origin-cloud-event-proxy:4.14
                 - name: WATCH_NAMESPACE
-                  value: ""
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
```

Comment:
did you run `make bundle`?  This should be changed back to `value: ""` after running `make bundle`, otherwise the ptp-operator would not be able to detect amq interconnect status.